Beheerlandschap.md

```diff
@@ -23,36 +25,17 @@ De relatie tussen deze omgevingen is gevisualiseerd in de onderstaande image:
 In GitHub bestaat voor elke API een repository maar ook voor enkele andere componenten. Zo hebben we nog een repository voor 'Zaken volgens GEMMA 2.0' dat dient als portaal naar alle gerelateerde GitHub repositories.
 Daarnaast bestaan er ook nog repositories voor de API testvoorziening, voor een tutorial over het werken met GitHub en voor API Beheer (waar dit document deel van uitmaakt).
 
-Het daadwerkelijk vervaardigen van de API standaarden en coderen van de Referentie Implementaties gebeurt in de GitHub repositories van de API's.
```

Comment:
Ik vind 'Coderen van de Referentie Implementaties' niet zo goed gekozen. Ik snap wat je bedoeld. Maar het is toch: Het realiseren van Referentie Implementaties? Of het bouwen van Referentie Implementaties?

Reply:
Prima om dit te verwoorden zoals jij voorstelt. Ik zal het zo wijzigen.